### PR TITLE
chore(bazel): Bazelify Button Group

### PR DESCRIPTION
**Pull request by furtnerl**
### <strong>Pull Request</strong>

<hr>
Hi, thank you for contributing to Barista with this pull request (PR).

To ensure a fast process and merging of your PR please make sure it fulfills the
coding standards and contribution guidelines.

- A feature proposal has been provided, discussed and approved first.
- There is a meaningful description of the issue in GitHub (Screenshots are
  often helpful).
- If the PR introduces breaking-changes or deprecations it matches the following
  guidelines.
  - The commit message follows our commit guidelines.
  - Tests for the changes have been added (for bug fixes / features).
  - Docs have been added / updated (for bug fixes / features).

Please choose the type appropriate for the changes below: <br>

#### Type of PR

<!-- Feature (non-breaking change which adds functionality)

#### Checklist

- [x] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: libs/barista-components/button-group/BUILD.bazel [hunk 1]
```diff
@@ -16,7 +41,58 @@ filegroup(
...
+    name = "button-group-styles",
+    src = "src/button-group.scss",
+    deps = [":scss_lib"],
+)
+
+sass_binary(
+    name = "button-group-item-styles",
+    src = "src/button-group-item.scss",
+    deps = [":scss_lib"],
+)
```
Comment (lukasholzer) [inline libs/barista-components/button-group/BUILD.bazel:61]: Use a multi sass binary for the styles

```suggestion
multi_sass_binary(
    name = "styles",
    srcs = [
        "src/button-group.scss",
        "src/button-group-item.scss",
        ":theme",
    ],
)
```

File: libs/barista-components/button-group/BUILD.bazel [hunk 2]
```diff
@@ -16,7 +41,58 @@ filegroup(
     ]),
 )
 
+sass_library(
+    name = "scss_lib",
```
Comment (lukasholzer) [inline libs/barista-components/button-group/BUILD.bazel:45]: I would rename to `theme` as it is more descriptive what the lib contains

```suggestion
    name = "theme",
```

**Comment by lukasholzer** [comment]
@furtnerl 

image

Strange that only the button-group got cached should have cache hits on the others as well. (remote cache hits)

**Comment by furtnerl** [comment]
> @furtnerl
> 
> image
> 
> Strange that only the button-group got cached should have cache hits on the others as well. (remote cache hits)

yep, I plan to looker closer into the remote build cache once I have gotten my PRs in, thanks for pointing out!

**Pull request merged**